specs/core/0_beacon-chain.md

```diff
@@ -1489,7 +1507,6 @@ For each `casper_slashing` in `block.body.casper_slashings`:
 
 * Let `slashable_vote_data_1 = casper_slashing.slashable_vote_data_1`.
 * Let `slashable_vote_data_2 = casper_slashing.slashable_vote_data_2`.
-* Let `indices(slashable_vote_data) = slashable_vote_data.custody_bit_0_indices + slashable_vote_data.custody_bit_1_indices`.
```

Comment:
One edge case here is that the same index can be contained in both `custody_bit_0_indices` and `custody_bit_1_indices`. I don't think anything breaks but wanted to double check.

On second thought, I think there is something cleaner than `custody_bit_0_indices` and `custody_bit_1_indices`. We have instead `validator_indices` (sorted for compression, as discussed in the Telegram chat) and an accompanying `custody_bitfield` (corresponding to the sorted indices). This is both more consistent with the rest of the spec, and removes the edge case above.

@djrtwo, @vbuterin.

Reply:
Yeah, I would support that.